src/docker/composeappengine.cc

```diff
@@ -42,11 +42,7 @@ bool ComposeAppEngine::fetch(const App& app) {
 
```

Comment:
for all the code above this like
```
if (!download(app)) {
      state.setState(AppState::State::kDownloadFailed);
      return result;
}
```
Should we set `result.err` to something more descriptive?

Reply:
download() should return just void since it throws an error if any error. For verify() and pullImages() we really don't know what's the error reason since we don't capture stderr of `docker-compose` process yet. Once, I implement the stderr capturing I'll update this code correspondingly. Either all these three methods will throw exception or return something like tuple<bool, errMsg>.